string id
geometry_msgs/Point target_point
geometry_msgs/Point box_tl
geometry_msgs/Point box_br